migration/003_add_deleted_at_column.sql

```diff
@@ -0,0 +1,7 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+ALTER TABLE "user" ADD COLUMN deleted_at TIMESTAMP WITH TIME ZONE DEFAULT NULL;
```

Comment:
Can you  change  to the `TIMESTAMP WITH TIME ZONE` in the first migration file at the `created_at` field ?

Reply:
@Brigant is a devil. There is no way back 😄

Reply:
Yea , I'm trying to fix my mistake using others hands :smiling_imp: But in general I want to fix it as soon as possible :innocent:

Reply:
Fixing migrations that are already in the main branch is not entirely correct. This also needs to be taken out as a separate small task, which will be done as the 4th migration as a separate pull request